doc/PLUGINS.md

```diff
@@ -200,6 +200,20 @@ corresponding payloads are listed below.
 
 ### Notification Types
 
+#### `channelopened`
+
+A notification for topic `channelopened` is sent if a peer successfully funded a channel
```

Comment:
We seem to get inconsistent with the naming of hooks:

 - `htlc_accepted`
 - `invoice_payment`
 - `db_write`
 - `channelopened`

I'd suggest adding the `_` for consistency's sake :-)

Reply:
Ok :-)